### PR TITLE
Pass kwargs to get OpenSearch client from_texts

### PR DESCRIPTION
**Pull request by naveentatikonda**
### Description
Pass kwargs to get OpenSearch client from `from_texts` function

### Issues Resolved
https://github.com/hwchase17/langchain/issues/2819

### CODE REVIEW

File: langchain/vectorstores/opensearch_vector_search.py [hunk 1]
```diff
@@ -428,7 +428,10 @@ def from_texts(
         opensearch_url = get_from_dict_or_env(
             kwargs, "opensearch_url", "OPENSEARCH_URL"
         )
-        client = _get_opensearch_client(opensearch_url)
+        # Remove the opensearch_url from kwargs to avoid passing
+        # the value twice for argument opensearch_url
+        kwargs.pop("opensearch_url")
+        client = _get_opensearch_client(opensearch_url, **kwargs)
```
Comment (hwchase17) [inline langchain/vectorstores/opensearch_vector_search.py:434]: what about all the other kwargs below?

Comment (naveentatikonda) [reply langchain/vectorstores/opensearch_vector_search.py:434]: Sorry, didn't get your question?

Comment (hwchase17) [reply langchain/vectorstores/opensearch_vector_search.py:434]: there are a lot of other arguments in "kwargs" that we use below. do we want to pop them as well? or are they relevant for the client

Comment (naveentatikonda) [reply langchain/vectorstores/opensearch_vector_search.py]: Got it. I have updated the code to pop all those arguments before getting the opensearch client.

**Review by hwchase17 — APPROVED** [review]
thanks!

**Pull request merged**